build.gradle.kts

```diff
@@ -35,6 +35,8 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-data-jpa")
     implementation("org.springframework.boot:spring-boot-starter-validation")
     implementation("org.springframework.boot:spring-boot-starter-actuator")
+    implementation("org.springframework.boot:spring-boot-starter-security")
+    implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
```

Comment:
I teamet har vi så langt kun brukt token-support for tokenautentisering. Jeg synes vi burde fortsette med det for å ikke være inkonsekvent. Token-support validerer også på alle nødvendige claims i tokenet som Nais krever, og er ellers like enkel i bruk som valideringen fra disse pakkene.

Reply:
Å skaffe OBO-tokens etc. er også enkelt.